spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -357,37 +357,19 @@
   end
 
   describe '#aal' do
-    context 'when DEFAULT_AAL passed' do
-      before do
-        default = Saml::Idp::Constants::DEFAULT_AAL_AUTHN_CONTEXT_CLASSREF
-        IdentityConfig.store.valid_authn_contexts.push(default)
-      end
-
-      after do
-        IdentityConfig.store.valid_authn_contexts.pop
-      end
-
-      let(:acr_values) { Saml::Idp::Constants::DEFAULT_AAL_AUTHN_CONTEXT_CLASSREF }
+    context 'when no AAL passed' do
+      let(:acr_values) { Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF }
 
       it 'returns 0' do
-        expect(form.aal).to eq(Saml::Idp::Constants::DEFAULT_AAL_AUTHN_CONTEXT_CLASSREF)
+        expect(form.aal).to eq(0)
```

Comment:
**question:** should this return `urn:gov:gsa:ac:classes:sp:PasswordProtectedTransport:duo` instead of 0?

Reply:
`aal` is an integer in the database, so I think this may be a bug in the current implementation to have the form return a string